lively.merger/tests/differ-test.js

```diff
@@ -0,0 +1,106 @@
+/* global describe, it, beforeEach, afterEach */
+import { expect } from 'mocha-es6';
+import { Differ } from '../differ.js';
+import { Morph, Ellipse } from 'lively.morphic';
+import { Color } from 'lively.graphics';
+
+function isEmpty (obj) {
+  for (let x in obj) {
+    if (obj.hasOwnProperty(x)) {
+      return false;
+    }
+  }
+  return true;
+}
+
+describe('lively.merger >> Differ', () => {
+  let morph1, morph2;
+  beforeEach(() => {
+    morph1 = new Morph();
+    morph2 = new Morph();
+
+    morph1.openInWorld();
+    morph2.openInWorld();
+  });
+
+  afterEach(() => {
+    morph1.abandon();
+    morph2.abandon();
+  });
+
+  describe('#diffMorphsWithIds', () => {
+    it('detects if the morphs with the first ID is not alive', () => {
+      expect(() => {
+        Differ.diffMorphsWithIds('morph1id', 'morph2id');
+      }).to.throw('Cannot diff morphs, morph1 with id morph1id not found');
+    });
+
+    it('detects if the morphs with the second ID is not alive', () => {
+      expect(() => {
+        Differ.diffMorphsWithIds(morph1.id, 'morph2id');
+      }).to.throw('Cannot diff morphs, morph2 with id morph2id not found');
+    });
+
+    it('runs without error if both morphs are alive', () => {
```

Comment:
I see why we would like to test this, but I have the feeling even if it doesn't throw an error it doesn't mean it works properly

Reply:
Think we should check the result of the diff

Reply:
Hmm, fair. I tried to test the result edge-cases (like empty differingProperties if the object is the same), but really testing for the result is somewhat hard, as the function only works on morphs and their properties are really large... Any idea how to get around that?

Reply:
I mean essentially we are testing diffMophs here, which we are testing below again.
Maybe we could just delete this test because it is not needed?

Reply:
fair enough